open_flamingo/eval/evaluate.py

```diff
@@ -452,71 +509,71 @@ def get_prompt(sample):
 
 
 def evaluate_vqa(
+    args,
     eval_model,
-    batch_size,
-    image_dir_path,
-    questions_json_path,
-    annotations_json_path,
     seed=42,
     max_generation_length=5,
```

Comment:
same comments as above re:generation hyperparameters, especially because these differ by dataset it would be great to make sure we're either using standard default values or setting these explicitly via a flag